TrkHitReco/fcl/prolog_trigger.fcl

```diff
@@ -5,57 +5,52 @@ BEGIN_PROLOG
 
 # flag hits from low-energy electrons (Compton electrons, delta rays, ...)
 # First, configure the clusters
-# Original 2-level clusterer
-TLTClusterer : { 
-    MaxNIterations : 10
-    TimeDifference : 25.0
-    MaxDistance : 100.0
-    HitDistance : 5.0
-    SeedDistance : 25.0
-    MaxTimeDifference : 40.0
-    BackgroundMask : []
-    SignalMask : ["TimeSelection", "EnergySelection","RadiusSelection"]
-}
-# Bertrand's faster version
 TNTClusterer : { 
-    MaxNIterations : 10
-    TimeDifference : 25.0
-    MaxDistance : 100.0
-    HitDistance : 5.0
-    SeedDistance : 25.0
-    MaxTimeDifference : 40.0
-    BackgroundMask : []
-    SignalMask : ["TimeSelection", "EnergySelection","RadiusSelection"]
+    HitDistance      : 5.0
+    SeedDistance     : 20.0
+    ClusterDiameter  : 5.0
+    ClusterTime      : 20.0
+    MaxHitTimeDiff   : 30.0
+    MaxSumDistance   : 100.0
+    MaxCluIterations : 10
+    MaxDistance      : 100.0
+    MinHitError      : 5.0
+    TimeRMS          : 2.0
+    DeltaTimeBinMin  : 10
+    MedianCentroid   : false
+    ComboInit        : true
+    TestFlag         : true
```

Comment:
Why "TestFlag" is set to true?

Reply:
Testflag tells the clustering algorithm to check the signal / background flag for the hits to use in the clustering. If you want to look at all hits for any reason, you need to set it to false, but it should be true by default